### PR TITLE
delete EM6

### PR DESCRIPTION
**Pull request by Yizhuzhu**

### CODE REVIEW

File: web/yo/app/scripts/controllers/drugs.js [hunk 1]
```diff
@@ -71,7 +75,7 @@ angular.module('oncokbApp')
 
             $scope.removeDrug = function (drug) {
                 if ($scope.drugMap[drug.uuid]) {
-                    var genes = _.map($scope.drugMap[drug.uuid], (gene) => {
+                    var genes = _.map($scope.drugMap[drug.uuid], function(gene){
```
Comment (victoria34) [inline web/yo/app/scripts/controllers/drugs.js]: I think you can code in this way:
var genes = _.map($scope.drugMap[drug.uuid], 'geneName');
https://lodash.com/docs/#map

File: web/yo/app/scripts/controllers/drugs.js [hunk 2]
```diff
@@ -42,7 +42,11 @@ angular.module('oncokbApp')
 
 
             function hasSameName(newDrugName, uuid) {
-                return _.some(mainUtils.getKeysWithoutFirebasePrefix($scope.drugList), (key) => ($scope.drugList[key].uuid !== uuid && (newDrugName === $scope.drugList[key].drugName || $scope.drugList[key].synonyms !== undefined && $scope.drugList[key].synonyms.indexOf(newDrugName) > -1)) === true);
+                return _.some(mainUtils.getKeysWithoutFirebasePrefix($scope.drugList), function(key){
+                    if(($scope.drugList[key].uuid !== uuid && (newDrugName === $scope.drugList[key].drugName || $scope.drugList[key].synonyms !== undefined && $scope.drugList[key].synonyms.indexOf(newDrugName) > -1)) === true){
```
Comment (victoria34) [inline web/yo/app/scripts/controllers/drugs.js]: What if it returns `undefined`? What will happen in curation platform?

Comment (Yizhuzhu) [reply web/yo/app/scripts/controllers/drugs.js]: It will be handled in $scope.saveDrugName. undefined means can't find any.

Comment (victoria34) [reply web/yo/app/scripts/controllers/drugs.js]: `Undefined` is quite easy to cause errors. Could you just return an empty string?
```
return $scope.drugList[key].uuid !== uuid && (newDrugName === $scope.drugList[key].drugName || $scope.drugList[key].synonyms !== undefined && $scope.drugList[key].synonyms.indexOf(newDrugName) > -1) ? key : '';
```

Comment (Yizhuzhu) [reply web/yo/app/scripts/controllers/drugs.js]: Sorry, I didn't explain it clearly.  The result returned will be the result of _.some function. According to the Lodash document, "Returns true if any element passes the predicate check, else false".

**Pull request merged**